arcgis-ios-sdk-samples/Analysis/Viewshed (location)/ViewshedLocation.storyboard

```diff
@@ -1,42 +1,58 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It seems like there would be a lot of value in making a reusable cell subclass for each type of cell:

- The cell would only need to be designed once.
- It would ensure a consistent look for each instance.
- The view controller would only need a single outlet for each cell.

Reply:
In previous samples you recommended using static cells for settings. Maybe prototype cells make more sense here since there is lots of repetition, but shouldn't we be consistent? In any case, I'm not going to change it for this sample since it's already hooked up manually.

Reply:
You can use a `UITableViewCell` subclass with static cells.

Reply:
Ohhhh now I see what you're saying. Interface builder is still the land of many secrets to me. I'll look into it.

Reply:
You should be able to make the cells designable, but designables seem to not compile successfully. I suspect it has to do with how we specify the path the `ArcGIS`. Now that Xcode 10.1 is out of beta, I'll submit a PR that updates the path to the framework and uses the new build system. That may fix the problem.